pkgs/by-name/fi/fichub-cli/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "fichub-cli";
+  version = "0.9.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-j/+VgiAgJuJJZlBiY/C//cyYELa/+z6pGkqL+Gi8pLE=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    platformdirs
+    beautifulsoup4
+    click
+    click-plugins
+    colorama
+    loguru
+    requests
+    tqdm
+    typer
+  ];
+
```

Comment:
Can we add a `pythonImportsCheck` somehow?

Reply:
The package includes tests, so I didn't think it was necessary

Reply:
It is always necessary to add them.

Also please call the tests explicitly preferable with pytest.

Reply:
I used `pytestCheckHook` instead of explicitly calling `pytest` — is this okay?

Reply:
yeah, that should always be used